### PR TITLE
Set Heroku configurations

### PR DESCRIPTION
**Pull request by laurajaime**
- Procfile with Sidekiq and migrations
- Puma configuration
- Sidekiq configuration

### CODE REVIEW

File: config/initializers/figaro.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
 env = Rails.env
-keys = %w(SECRET_KEY_BASE)
```
Comment (tramuntanal) [inline config/initializers/figaro.rb:4]: don't we  need a different secret-key in each environment?

File: spec/system/homepage_spec.rb [hunk 2]
```diff
@@ -8,12 +8,12 @@
   let!(:organization) do
     create(
       :organization,
-      name: "Decidim Application",
```
Comment (tramuntanal) [inline spec/system/homepage_spec.rb:11]: I would keep this test as is, without any modification, unless we have customizations, which is not the case

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Great job @LauraJaimeC !

I've just left a couple of coments to check

**Comment by laurajaime** [comment]
🎉

**Review by tramuntanal — APPROVED** [review]

**Pull request merged**